docs/Doxyfile.in

```diff
@@ -481,13 +468,13 @@ FILE_VERSION_FILTER    =
 # The QUIET tag can be used to turn on/off the messages that are generated 
 # by doxygen. Possible values are YES and NO. If left blank NO is used.
 
-QUIET                  = NO
+QUIET                  = YES
 
 # The WARNINGS tag can be used to turn on/off the warning messages that are 
 # generated by doxygen. Possible values are YES and NO. If left blank 
 # NO is used.
 
-WARNINGS               = YES
+WARNINGS               = NO
```

Comment:
wouldn't it be better to fix the doxygen docs rather than hide the warnings? I am afraid if we disable warnings here we will never fix doxygen syntax errors.

Reply:
Yes, I agree. It would be better. But there are thousands of them.

Reply:
So, let's revert this change and start fixing Doxgen Syntax errors in a separate PR!